src/metabase/query_processor/store.clj

```diff
@@ -178,8 +178,23 @@
   [database-id-or-metadata-provider & body]
   `(do-with-metadata-provider ~database-id-or-metadata-provider (^:once fn* [] ~@body)))
 
+(defn- pretty-metadata-error-msg
+  "Pretty error message for eg. [[missing-bulk-metadata-error]]."
+  [metadata-type id]
+  (let [metadata-type-str (pr-str metadata-type)
+        id-str (pr-str id)]
+    (case metadata-type
+      :metadata/card    (tru "Failed to fetch Card {0}." id-str)
+      :metadata/column  (tru "Failed to fetch Field {0}: Field does not exist or belongs to different Database."
```

Comment:
Switching from fields to columns might become a source of problems. Not all columns are fields.